AppCenter/AppCenter/Internals/Util/MSUtility.m

```diff
@@ -39,4 +39,28 @@ + (NSString *)sdkVersion {
   return [NSString stringWithUTF8String:appcenter_library_info.ms_version];
 }
 
++ (NSObject *)unarchiveKeyedData:(NSData *)data {
+  if (@available(iOS 11.0, macOS 10.13, watchOS 4.0, *)) {
+    NSKeyedUnarchiver *unarchiver = [[NSKeyedUnarchiver alloc] initForReadingFromData:data error:nil];
```

Comment:
I'd probably capture the error and print it even if it's supposed to work.

Reply:
That's fixed in another bug right now but not related to feature I guess

Reply:
AB#80422